index.html

```diff
@@ -1201,12 +1205,14 @@ <h3>Getting Started</h3>
         <p>
 Once a developer has prototyped their credential to a point where they believe
 all of the credential properties are stable, it is advised that they generate
-vocabulary and context files for their application and publish them at stable URLs
-so that other developers can use the same vocabulary and context to achieve
-interoperability. This process is covered in Section
-<a href="#extensibility"></a>. Alternatively, developers can reuse existing vocabulary
-and context files that happen to fit their use case. They can explore the
-[[[VC-SPECS]]] [[VC-SPECS]] for reusable resources.
+vocabulary and context files for their application and publish them at stable
+URLs, so that other developers can use the same vocabulary and context to achieve
+interoperability. The `https://www.w3.org/ns/credentials/examples/v2` URL above
+would then be replaced with the URL of an application-specific context. This
```

Comment:
I can live with "application-specific" but I think "use-case-specific" is better:

```suggestion
would then be replaced with the URL of an use-case-specific context. This
```

Reply:
Because, what would a change request be without a nitpick? `a` instead of `an`, here.
```suggestion
would then be replaced with the URL of a use-case-specific context. This
```